types/dragino/lsn50v2-d23/weight.schema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$id": "https://akenza.io/dragino/lsn50v2-d23/weight.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Weight",
+  "type": "object",
+  "properties": {
+    "weight": {
+      "title": "Weight",
+      "type": "number",
+      "description": "Weight "
```

Comment:
what is the unit?

Reply:
Can not find a unit in the documentation. It is probably dependent on the connected sensor.